src/kili/services/json_response/label.py

```diff
@@ -0,0 +1,26 @@
+"""Module for label parsing."""
+
+from typing import Dict
+
+from .json_response import ParsedJobs
+
+
+class ParsedLabel(Dict):
+    """Class that parses a label."""
+
+    def __init__(self, label: Dict, json_interface: Dict, inplace: bool = False):
```

Comment:
not sure we should let the choice to the user. It is in general a bad practice to do it in place.